tests/unit_tests/cpu/core_detector.cpp

```diff
@@ -601,12 +610,14 @@ GTEST_TEST(detray_tools, detector_volume_construction) {
             << "error at index: " << idx;
     }
 
-    // check that the transform indices are continuous
+    // check that the transform indices are continuous for the newly added
+    // surfaces. The first new transform belongs to the volume itself
     for (std::size_t idx :
-         detray::views::iota(d.transform_store().size() - 1)) {
+         detray::views::iota(dindex_range{3, d.n_surfaces()})) {
```

Comment:
3 not 1?

Reply:
Yes, for the test the detector is pre-filled with some surfaces, so the index of the surfaces that were added by the volume builder starts at 3